### PR TITLE
Add page about technologies and their proven versions

### PR DESCRIPTION
**Pull request by lucendio**
## Checklist:

Please tick the following before making your PR:

* [X] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [X] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

**Comment by lucendio** [comment]
__Side note:__ In a recent conversation @jschaul pointed out that such a list should not be put together by hand. Ideally, we want to obtain those version in an automated fashion, but frankly, we are not there yet. Meaniing, these numbers are scattered all over the place. So, while aiming for this in the feature, this small curated list is still an improvement over what was there before.

File: src/how-to/install/version-requirements.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Ubuntu: 18.04
+- Docker: latest
+- Kubernetes: 1.14.2 (tested: Kubspray `e2f5a97 <https://github.com/kubernetes-sigs/kubespray/commit/e2f5a9748e4dbfe2fdba7931198b0b5f1f4bdc7e>`_)
+
+
+Automation
+----------
+
+- Ansible: 2.7
+- Helm: >= v2
```
Comment (jschaul) [inline src/how-to/install/version-requirements.rst:30]: Can we bump this to helm 3?

Comment (lucendio) [reply src/how-to/install/version-requirements.rst:30]: As minimal required version you mean?

Comment (jschaul) [reply src/how-to/install/version-requirements.rst:30]: yes.

Comment (lucendio) [reply src/how-to/install/version-requirements.rst:30]: Actually, why not. Stop carrying tiller with us is a good thing.

**Pull request merged**